sql/moz-fx-data-shared-prod/telemetry_derived/desktop_retention_clients_v1/query.sql

```diff
@@ -29,18 +29,16 @@ new_profiles AS (
     attribution_dlsource,
     attribution_medium,
     attribution_ua,
+    attribution_experiment,
     distribution_id,
     -- cfs.isp_name,
     cfs.normalized_channel,
     startup_profile_selection_reason,
-    attribution_experiment,
-    -- mozfun.norm.os(cfs.os) AS normalized_os,
-    cfs.normalized_os,
+    mozfun.norm.os(cfs.normalized_os) AS normalized_os,
```

Comment:
this field is not properly normalized on first_seen right now so I'm passing it through the UDF.
Also removing the normalized_os_version since it doesn't match with our calculation and until fixed I think it's better to leave NULL for the first-seen only clients than having weird values in our dashboards.